css/font-awesome.css

```diff
@@ -61,7 +61,7 @@
 }
 .fa-border {
   padding: .2em .25em .15em;
-  border: solid 0.08em #eeeeee;
+  border: solid 0.08em #eee;
```

Comment:
it looks like an unwanted change, maybe a different version of the less compiler?

There is no difference at all, but changes to the main .css could be removed from this PR to keep it clean